startup.py

```diff
@@ -21,6 +21,6 @@
     elif choice == '2':
         dbmanager.stop()
     elif choice == '3':
-        initdb.main("","")
+        initdb.main()
```

Comment:
@kevintaing86 the initdb module now throws exceptions which can be caught and printed by: 

`try:
       initdb.main()
except Exception as err:
       print(err)`